applications_tests/cfd_dem_coupling_3d/CMakeLists.txt

```diff
@@ -10,11 +10,19 @@ file(COPY particle_sedimentation_files/dem.triangulation DESTINATION ${CMAKE_CUR
 file(COPY particle_sedimentation_files/dem.triangulation.info DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/particle_sedimentation.${CMAKE_LOWER_BUILD_TYPE})
 file(COPY particle_sedimentation_files/dem.triangulation_fixed.data DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/particle_sedimentation.${CMAKE_LOWER_BUILD_TYPE})
 file(COPY particle_sedimentation_files/dem.triangulation_variable.data DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/particle_sedimentation.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.particles DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.pvdhandler DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.simulationcontrol DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.triangulation DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.triangulation.info DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.triangulation_fixed.data DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
+file(COPY fluidized_bed_files/dem.triangulation_variable.data DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/fluidized_bed.${CMAKE_LOWER_BUILD_TYPE})
 
 
 DEAL_II_PICKUP_TESTS()
 
 if (CMAKE_BUILD_TYPE STREQUAL "Debug" )
        	set_tests_properties(cfd_dem_coupling_3d/particle_sedimentation.debug PROPERTIES TIMEOUT 1200) 
+       	set_tests_properties(cfd_dem_coupling_3d/fluidized_bed.debug PROPERTIES TIMEOUT 2000)
```

Comment:
It's always a concern when tests are becoming this long. Is it possible to design a shorter one?